src/main/java/mate/model/Car.java

```diff
@@ -9,6 +10,10 @@ public class Car {
     private Manufacturer manufacturer;
     private List<Driver> drivers;
 
+    public Car() {
+        this.drivers = new ArrayList<>();
```

Comment:
```suggestion
        drivers = new ArrayList<>();
```

do we need `this` keyword here?